### PR TITLE
Issue/87 - Unable to hide Coupons tab with AffiliateWP v2.6

### PR DESCRIPTION
**Pull request by tubiz**
Issue: #87

### CODE REVIEW

File: includes/class-functions.php [hunk 1]
```diff
@@ -38,14 +38,15 @@ public function is_default_tab( $tab_slug ) {
...
-			'settings'  => __( 'Settings', 'affiliate-wp' )
+			'urls'      => __( 'Affiliate URLs', 'affiliatewp-affiliate-area-tabs' ),
+			'stats'     => __( 'Statistics', 'affiliatewp-affiliate-area-tabs' ),
+			'graphs'    => __( 'Graphs', 'affiliatewp-affiliate-area-tabs' ),
+			'referrals' => __( 'Referrals', 'affiliatewp-affiliate-area-tabs' ),
+			'payouts'   => __( 'Payouts', 'affiliatewp-affiliate-area-tabs' ),
+			'visits'    => __( 'Visits', 'affiliatewp-affiliate-area-tabs' ),
+			'coupons'   => __( 'Coupons', 'affiliatewp-affiliate-area-tabs' ),
+			'creatives' => __( 'Creatives', 'affiliatewp-affiliate-area-tabs' ),
+			'settings'  => __( 'Settings', 'affiliatewp-affiliate-area-tabs' ),
```
Comment (amdrew) [inline includes/class-functions.php:49]: @tubiz Any particular reason why the text domains were changed? 

I can't recall the exact reason why they were set to `affiliate-wp` in the first place but I believe it was to make it as seamless as possible. I.e. a customer who had translated `urls` (or had a translation pack) would not need to translate it again after installing this add-on. Changing the translations may mean customers losing their existing  translations.

**Review by amdrew — CHANGES_REQUESTED** [review]
@tubiz Works great! Left a small comment on text domains.

File: includes/class-functions.php (hunk 1 above)
Comment (tubiz) [reply includes/class-functions.php:49]: @amdrew no reason, I wasn't too sure about changing it too. Something was telling me it was this way for a reason.

I undo the change thrice before I finally commited it. I guess I should have listen to my instincts then.

I will revert the change.

**Comment by tubiz** [comment]
@amdrew PR updated.

**Review by amdrew — APPROVED** [review]

**Comment by amdrew** [comment]
@tubiz Thanks! Once @DrewAPicture reviews/approves, could you also please push out a release?

**Review by DrewAPicture — APPROVED** [review]

**Comment by tubiz** [comment]
@amdrew @DrewAPicture I don't have push access to the plugin on WP.org

**Comment by DrewAPicture** [comment]
@tubiz I've added you as a committer.

**Comment by tubiz** [comment]
Thanks @DrewAPicture

**Pull request merged**